### PR TITLE
#169817809 Integrating Codeclimate Test Coveralls

### PR DESCRIPTION
**Pull request by Jajabenit250**
### What does this PR do?
-  Integrating Codeclimate Test Coveralls
#### Description of Task to be completed?
- Adding Coveralls badge on Readme
- get code coverage on every commit made
#### What are the relevant pivotal tracker stories?
[#169817809](https://www.pivotaltracker.com/story/show/169817809)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,5 @@ node_modules
 dist
 package-lock.json
 coverage
+./package-lock.json
```
Comment (Ruqoyah) [inline .gitignore]: Why did you add this?

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,5 @@ node_modules
...
+.env
```
Comment (Ruqoyah) [inline .gitignore]: Is there anything you added to .env? If yes, you need to add `.env.example` file that contains an explanation of what's supposed to be added e.g `DB_USERNAME=your database username`

Comment (Jajabenit250) [reply .gitignore]: @Ruqoyah  solved.
